codec_context_test.go

```diff
@@ -121,6 +125,10 @@ func TestCodecContext(t *testing.T) {
 	require.Equal(t, NewRational(15, 1), cc4.TimeBase())
 	require.Equal(t, 16, cc4.Width())
 	require.Equal(t, 4, cc4.ExtraHardwareFrames())
+	require.Equal(t, 1, cc4.MaxBFrames())
+	require.Equal(t, 1_5000_000, cc4.RcMaxRate())
```

Comment:
Test are failing here since `1_5000_000` should be casted as an `int64`: `int64(1_5000_000)`